src/network/networkSettings.js

```diff
@@ -1,4 +1,6 @@
 export const LOCAL_ID = 9999999999
```

Comment:
please also review names of all _network/*_ files – capital letter at the start of the name is permitted only for files, which exports class by default

there is the _hdkey_ and _hdwallet_ variables inside of the _MnemonicProvider.js_ that you should rename using lower camel-case rule as well. To prevent such errors in the future, use the automatic IDE typo checker